index.html

```diff
@@ -64,8 +64,31 @@
     <h2>
       Introduction
     </h2>
+    <p>
+      This document defines an API enabling a website to request presentation of a <dfn data-abbr>verifiable digital credential</dfn> (VDC).
+    </p>
+    <p>
+      The API design is agnostic to both credential presentation exchange protocols and credential formats. However, this document includes a [[[#protocol-registry]]] to ensure interoperability.
```

Comment:
```suggestion
      The API design is agnostic to both credential presentation exchange protocols and credential formats. However, to promote interoperability this document includes a [[[#protocol-registry]]].
```

IMHO we're very far from being able to "ensure interoperability" 😄